cirq/google/engine/engine_processor.py

```diff
@@ -277,29 +277,65 @@ def list_reservations(
 
     def get_schedule(
             self,
-            from_time: datetime.datetime = datetime.datetime.now(tz=utc),
-            to_time: datetime.datetime = datetime.datetime.now(tz=utc) +
-            datetime.timedelta(weeks=2),
-            time_slot_type: qenums.QuantumTimeSlot.TimeSlotType = None
+            from_time: Union[None, datetime.datetime, datetime.
+                             timedelta] = datetime.timedelta(),
+            to_time: Union[None, datetime.datetime, datetime.
+                           timedelta] = datetime.timedelta(weeks=2),
+            time_slot_type: Optional[qenums.QuantumTimeSlot.TimeSlotType] = None
     ) -> List[EngineTimeSlot]:
         """Retrieves the schedule for a processor.
 
         The schedule may be filtered by time.
 
         Time slot type will be supported in the future.
+
+        Args:
+            from_time: Filters the returned scheduled to only include entries that end no earlier than the given value.
+                Specified either as an absolute time (datetime.datetime) or as a time relative to now
+                (datetime.timedelta). Defaults to now (a relative time of 0). Set to None to omit this filter.
+            to_time: Filters the returned scheduled to only include entries that start no later than the given value.
```

Comment:
nit: s/scheduled/schedule/

Also applies to l.293 above.

Reply:
Done